rosdep/python.yaml

```diff
@@ -5296,6 +5296,16 @@ python-xmlplain-pip:
   ubuntu:
     pip:
       packages: [xmlplain]
+python-xmlschema-pip:
+  debian:
+    pip:
+      packages: [xmlschema]
+  fedora:
+    pip:
+      packages: [xmlschema]
+  ubuntu:
+    pip:
+      packages: [xmlschema]
```

Comment:
This is implicitly a Python 2 key.  Additionally, it looks like this package is available in Fedora, and also Debian as a system package.  That probably means it will eventually become a package in Ubuntu (though that is not guaranteed).

So with all of that said, my suggestion here is that we make this look more like the following:

```
python3-xmlschema:
  debian:
    '*': [python3-xmlschema]
    'buster':
      pip:
        packages: [xmlschema]
  fedora: [python3-xmlschema]
  ubuntu:
    pip:
      packages: [xmlschema]
```

Reply:
Thank you, I fixed this problem.